inst/extdata/correctionsataglance-example.json

```diff
@@ -103,267 +103,187 @@
     "type": "Discharge",
     "points": []
   },
-  "fieldVisitMeasurements": [
+  "corrections": [],
+  "reportMetadata": {
+    "country": "UNITED STATES OF AMERICA",
+    "altitude": "  657.64",
+    "requestingUser": "thongsav",
+    "agency": "USGS ",
+    "endDate": "2015-12-24T00:00:00-06:00",
+    "drainageArea": "    2840",
+    "requestDateTime": "2015-12-28T10:26:27.915-06:00",
+    "nwisRaAuthToken": "565e39ff-8e34-4e98-8170-6a3d6fe8b083",
+    "latitude": "37.9299166666667",
+    "county": "PHELPS",
+    "description": "Shows overview of correction information",
+    "title": "Corrections At a Glance",
+    "longitute": "-91.9773333333333",
+    "primaryParameter": "Discharge.ft^3/s@06933500",
+    "primaryTsIdentifier": "0f20e9ce98c646bb9c0d1439d184423d",
+    "requestId": "7450194a-69bd-411d-b0d6-6bbc46dc159d",
+    "contributingDrainageArea": "        ",
+    "altitudeDatumCode": "NGVD29    ",
+    "siteNumber": "06933500       ",
+    "stationName": "Gasconade River at Jerome, MO",
+    "coordinateDatumCode": "NAD83     ",
+    "state": "MISSOURI",
+    "startDate": "2014-12-24T00:00:00-06:00",
+    "stationId": "06933500"
+  },
+  "fieldVisits": [
     {
```

Comment:
Yes, this report is interested in top level field visits, not readings or measurements.